make_beam/get_delays_small.c

```diff
@@ -418,7 +442,12 @@ void get_delays(
 
                 // Now, calculate the inverse Jones matrix
                 if (invJi != NULL) {
-                    if (pol == 0) {
+                    if (pol == 0) { // This is just to avoid doing the same calculation twice
+                        // Apply parallactic angle correction if Hyperbeam was used
+                        //if (beam_model == BEAM_FEE2016) {
+                        //    mult2x2d_RxC( P, Ji, Ji );  // Ji = P x Ji (where 'x' is matrix multiplication)
+                        //}
```

Comment:
Why is this commented out? Might as well remove it if we don't need it

Reply:
There _will_ have to be a parallactic angle correction at some point. Something about the way it's implemented here is wrong, hence why it's commented out, but leave it here as a memo to ourselves to figure out how to apply the correction, uh, correctly.

Reply:
O I see, you haven't got to that bit yet.